src/Form/PasswordConstraintAddForm.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\password_enhancements\Form;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\password_enhancements\PasswordConstraintInterface;
+use Drupal\password_enhancements\PasswordConstraintManager;
+use Drupal\user\RoleInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides an add form for password constraints.
+ *
+ * @internal
+ */
+class PasswordConstraintAddForm extends PasswordConstraintFormBase {
```

Comment:
If it is `@internal` then it should be `@final` unless another class extends this in this module.